recipes/keras-tuner/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "keras-tuner" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f018ded5127e7c588bd78faa753fa1f4108f1f28c0e5694a06bfb1f08204215b"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - colorama
+    - future
+    - numpy
+    - pip
+    - python
+    - requests
+    - scikit-learn
+    - scipy
+    - tabulate
+    - terminaltables
+    - tqdm
+  run:
+    - colorama
+    - future
+    - numpy
+    - python
+    - requests
+    - scikit-learn
+    - scipy
+    - tabulate
+    - terminaltables
+    - tqdm
+
+test:
+  imports:
+    - kerastuner
+    - kerastuner.abstractions
+    - kerastuner.applications
+    - kerastuner.distribute
+    - kerastuner.engine
+    - kerastuner.oracles
+    - kerastuner.protos
+    - kerastuner.tuners
+
+about:
+  home: "https://github.com/keras-team/keras-tuner"
+  license: "Apache 2.0"
+  license_family: "APACHE"
+  license_file: "LICENSE"
```

Comment:
It looks like you will need to manually package the license file:

https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

Please raise an issue in the upstream repo asking them to include it and add the issue as a comment here.